pkg/api/unversioned/deep_copy_generated.go

```diff
@@ -21,127 +21,15 @@ limitations under the License.
 package unversioned
 
 import (
-	"time"
-
 	conversion "k8s.io/kubernetes/pkg/conversion"
+	time "time"
 )
 
-func DeepCopy_unversioned_APIGroup(in APIGroup, out *APIGroup, c *conversion.Cloner) error {
```

Comment:
The problem with the central place is that we already want to have smaller
client libraries - do they have to have all API groups added (with their
generated code) in order to consume?

Agree a lot of dependencies will be core, but we keep finding cases where
we suck in small dependencies from outside.  And as that group grows, we
also get more useless stuff.

On Mon, May 23, 2016 at 12:18 PM, Tim Hockin <notifications@github.com>
wrote:

> In pkg/api/unversioned/deep_copy_generated.go
> <https://github.com/kubernetes/kubernetes/pull/26051#discussion_r64246386>
> :
>
> >  )
> >
> > -func DeepCopy_unversioned_APIGroup(in APIGroup, out *APIGroup, c *conversion.Cloner) error {
>
> Frankly, I hate sprinkling this crap all over the place. I'd much rather
> code-gen into a central place, but that is maybe beyond scope for now.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26051/files/ee463c073a7f6853fc2a6019de435aa5d4e6c2c4#r64246386>
>